management/src/test/java/org/ehcache/management/providers/statistics/MissRatioTest.java

```diff
@@ -0,0 +1,173 @@
+    { newResourcePoolsBuilder().heap(1, MB).offheap(2, MB), statNamesOnHeapOffHeap, Arrays.asList(1l,2l,3l,4L) , Arrays.asList(1d,.25d), .25d },//4 heap misses, 1 offheap miss, 3 hits
+    { newResourcePoolsBuilder().heap(1, MB).offheap(2, MB), statNamesOnHeapOffHeap, Arrays.asList(4L,5L) , Arrays.asList(1d,1d), 1d },          //2 heap misses, 2 offheap misses, 0 hits
+
+
+    { newResourcePoolsBuilder().heap(1, MB).disk(2, MB), statNamesOnHeapDisk, Arrays.asList(1l,2l,3l) , Arrays.asList(1d,0d), 0d },       //3 heap misses, 0 disk misses, 3 hits
+    { newResourcePoolsBuilder().heap(1, MB).disk(2, MB), statNamesOnHeapDisk, Arrays.asList(1l,2l,3l,4L) , Arrays.asList(1d,.25d), .25d },//4 heap misses, 1 disk miss, 3 hits
+    { newResourcePoolsBuilder().heap(1, MB).disk(2, MB), statNamesOnHeapDisk, Arrays.asList(4L,5L) , Arrays.asList(1d,1d), 1d },          //2 heap misses, 2 disk misses, 0 hits
+
+    //offheap and disk configuration is not valid.  Throws IllegalStateException no Store.Provider found to handle configured resource types [offheap,disk]
+
+    //3 tiers
+    { newResourcePoolsBuilder().heap(1, MB).offheap(2, MB).disk(3, MB), statNamesThreeTiers, Arrays.asList(1l,2l,3l,1L) , Arrays.asList(.75d,1d,0d), 0d },         //3 heap misses, 3 offheap misses, 0 disk misses, 4 hits
+    { newResourcePoolsBuilder().heap(1, ENTRIES).offheap(2, MB).disk(3, MB), statNamesThreeTiers, Arrays.asList(1l,2l,2L,4L), Arrays.asList(.75d,1d,1d/3d), 1d/4d},//3 heap misses, 3 offheap misses, 1 disk miss, 3 hits
+    });
+
+  }
+
+  public MissRatioTest(Builder<? extends ResourcePools> resources, List<String> statNames, List<Long> getKeys, List<Double> tierExpectedValues, Double cacheExpectedValue) {
+    this.resources = resources.build();
+    this.statNames = statNames;
+    this.getKeys = getKeys;
+    this.tierExpectedValues = tierExpectedValues;
+    this.cacheExpectedValue = cacheExpectedValue;
+  }
+
+  @Test
+  public void test() throws InterruptedException, IOException {
+    DefaultManagementRegistryConfiguration registryConfiguration = new DefaultManagementRegistryConfiguration().setCacheManagerAlias("myCacheManager");
+    registryConfiguration.addConfiguration(new EhcacheStatisticsProviderConfiguration(1,TimeUnit.MINUTES,100,1,TimeUnit.MILLISECONDS,10,TimeUnit.MINUTES));
+    ManagementRegistryService managementRegistry = new DefaultManagementRegistryService(registryConfiguration);
+
+    Configuration cacheConfiguration = new DefaultConfiguration(EvictionTest.class.getClassLoader(),
+            new DefaultPersistenceConfiguration(diskPath.newFolder()));
+
+    Collection<Service> services = new ArrayList<Service>();
+    services.add(managementRegistry);
+
+    CacheManager cacheManager = null;
+
+    try {
+      cacheManager = new EhcacheManager(cacheConfiguration, services);
```

Comment:
You are right, that would make more sense since the management tests should use the api. I will change it.